install/kubernetes/helm/istio/README.md

```diff
@@ -38,31 +38,31 @@ The chart deploys pods that consume minimum resources as specified in the resour
 ## Installing the Chart
 
 1. If a service account has not already been installed for Tiller, install one:
-```
-$ kubectl apply -f install/kubernetes/helm/helm-service-account.yaml
-```
-
-2. Install Tiller on your cluster with the service account:
-```
-$ helm init --service-account tiller
-```
-
-3. Set and create the namespace where Istio was installed:
-```
-$ NAMESPACE=istio-system
-$ kubectl create ns $NAMESPACE
-```
-
-4. Install Istio’s [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
+   ```
+   $ kubectl apply -f install/kubernetes/helm/helm-service-account.yaml
+   ```
+
+1. Install Tiller on your cluster with the service account:
+   ```
+   $ helm init --service-account tiller
+   ```
+
+1. Set and create the namespace where Istio was installed:
+   ```
+   $ NAMESPACE=istio-system
+   $ kubectl create ns $NAMESPACE
+   ```
+
+1. Install Istio’s [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
```

Comment:
@sdake do you mind update here a bit by highlighting this is only required by the helm which < 2.10 , we have updated this in istio.io but not here yet https://preliminary.istio.io/docs/setup/kubernetes/helm-install/#installation-steps